build.gradle

```diff
@@ -44,12 +44,19 @@ configurations {
       // Force v4.5.0 of commons-collections4 to avoid CVEs in v4.4.0 from transitive dependecies:
       // CVE-2025-48924 (https://www.cve.org/CVERecord?id=CVE-2025-48924) and
       // CVE-2020-15250 (https://www.cve.org/CVERecord?id=CVE-2020-15250)
-      force "org.apache.commons:commons-collections4:4.5.0"
-
-      // Force v3.18 of commons-lang3 to avoid CVE-2025-48924
-      // (https://www.cve.org/CVERecord?id=CVE-2025-48924), without also
-      // upgrading ml-app-deployer to 6.0.0, which we are not ready to do yet.
-      force 'org.apache.commons:commons-lang3:3.18.0'
+      eachDependency { DependencyResolveDetails details ->
```

Comment:
You can hopefully get rid of these once you've upgraded to Kafka 4. Per the comment on the commons-lang3 one - that was due to an older version of ml-app-deployer, so it's likely that that "force" can be removed. Note as well, we never want to have these "useVersion" blocks in here without a corresponding `details.because` to document why we're forcing a version.